examples/vector_collection/vector_collection.py

```diff
@@ -0,0 +1,73 @@
+import logging
+
+from hazelcast import vector
+from hazelcast.core import HazelcastJsonValue
+from hazelcast.vector import Type, Vector
+
+logging.basicConfig(level=logging.DEBUG)
+
+from hazelcast.client import HazelcastClient, Config
+from hazelcast.proxy.vector_collection import Document
+
+
+def main():
+    # Connect to the cluster
+    cfg = Config()
+    cfg.cluster_members = ["localhost:5701"]
+    client = HazelcastClient(cfg)
+
+    vc_name = "my-vector-collection"
+
+    # Create the VectorCollection
+    indexes = [
+        vector.IndexConfig(name="default-vector", metric=vector.Metric.COSINE, dimension=2),
+    ]
+    client.create_vector_collection(vc_name, indexes=indexes)
+
+    # Use the VecotorCollection
+    vc = client.get_vector_collection(vc_name).blocking()
+    doc1 = Document(
+        "value1",
+        [
+            Vector("default-vector", Type.DENSE, [0.1, 0.5]),
+        ],
+    )
+
+    # Add the Document
+    key = "key-1"
+    vc.set(key, doc1)
+
+    # Add another Document
+    doc2 = Document(
+        "value2",
+        [
+            Vector("default-vector", Type.DENSE, [0.5, 0.7]),
+        ],
+    )
+
+    # Add the Document
+    key = "key-2"
+    vc.set(key, doc2)
+
+    # Search for a vector
+    results = vc.search_near_vector(
```

Comment:
do we have docstring/documentation of search_near_vector and other methods somewhere? I didn't see them.